models/shopify__order_lines.sql

```diff
@@ -58,8 +58,10 @@ with order_lines as (
         product_variants.option_1 as variant_option_1,
         product_variants.option_2 as variant_option_2,
         product_variants.option_3 as variant_option_3,
-        product_variants.tax_code as variant_tax_code,
-        product_variants.is_requiring_shipping as variant_is_requiring_shipping
+        product_variants.tax_code as variant_tax_code
+        {# ,
+        use inventoryitem or order line itself
```

Comment:
what is this line doing?

Reply:
just commenting out a deprecated field and making a note to bring it in from `inventoryitem` instead (for a later PR)

Reply:
Got it, since it's a note should we comment it out?

Reply:
ah yeah `{# ... #}` is the commenting syntax for sql-jinja! same as `/* ... */`, except this works for jinja as well

Reply:
got it! weird that the syntax in github makes it look like code though